src/app/pages/administration/system-maintenance/system-maintenance.jsx

```diff
@@ -33,17 +47,50 @@ const useStyles = makeStyles({
     alignItems: 'stretch',
     gap: '16px',
     [theme.breakpoints.up('md')]: {
-      display: 'grid',
-      gridTemplateColumns: '1fr 3fr',
+      display: 'flex',
```

Comment:
The default display value is `flex`. You shouldn't need to change the value because of a breakpoint check